app/controllers/schools/cohorts_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class Schools::CohortsController < Schools::BaseController
+  skip_after_action :verify_authorized
+  skip_after_action :verify_policy_scoped
+
+  def show
+    @school = current_user.induction_coordinator_profile.schools.first
+    @cohort = Cohort.find_by(start_year: params[:id])
+
+    @school_cohort = SchoolCohort.find_by(
```

Comment:
I wonder if we could just use the the id of @school_cohort as the  id in this request rather than start_year

Reply:
We could do, but in my opinion `/schools/cohort/2021` is a much nicer URL than `/schools/cohort/651d714f-a8d1-44ad-b424-c214f8a9f979`